spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -385,6 +391,9 @@ describe("kong.clustering.compat", function()
       end)
 
       teardown(function()
+        -- Truncate will clean all tables' records, which may cause some other tests to fail because the
+        -- number of records in the truncated table differs from the number of records after bootstrap.
+        -- So we need the above teardown to reset schema.
```

Comment:
I think that these comments should be in `lazy_teardown()`

Reply:
moved to the new `teardown()`.